client/app/bundles/course/assessment/submission/containers/QuestionGrade.jsx

```diff
@@ -77,14 +77,16 @@ class VisibleQuestionGrade extends Component {
             className="w-40"
             hiddenLabel
             inputProps={{ className: 'grade' }}
-            onBlur={(e) => this.processValue(e.target.value)}
+            onBlur={(e) => {
```

Comment:
You might not be able to use `useDebounce` here since it is a class component. If you want, you can convert it to functional component. I think it is a good practice. Do convert this, make this component tidier and use typescript.

Reply:
yeap.. already done, will push soon